LoopKit/DeviceManager/PumpManager.swift

```diff
@@ -203,4 +203,11 @@ public extension PumpManager {
             completion()
         }
     }
+    
+    /// Convenience wrapper for fetching current pump data without a completion
+    ///
+    func fetchCurrentPumpData() {
```

Comment:
Ah.  I see your point.  I didn't mean for this to be a default _implementation_ but rather a convenience method for _callers_ to not need to pass in a completion block (because not all callers care). (Because protocols can't have default parameter values).  I'll address this somehow...